src/js/main.js

```diff
@@ -0,0 +1,36 @@
+import auth from './auth';
+
+const authenticationPage = document.querySelector('#authentication-page');
+const signInButton = document.querySelector('#sign-in-button');
+const signOutButton = document.querySelector('#sign-out-button');
+const formButton = document.querySelector('#form-button');
+
+const authHandler = auth();
+
+const signIn = () => {
+  authenticationPage.classList.replace('flex', 'hidden');
+};
+
+const signOut = () => {
+  authenticationPage.classList.replace('hidden', 'flex');
+};
+
+const submitEvent = (event) => {
+  event.preventDefault();
+
+  signInButton.addEventListener('click', signIn, () => {
+    const emailInput = document.querySelector('#email');
+
+    const email = emailInput.value;
+    try {
+      authHandler.signIn(email);
+    } catch (error) {
+      error.message = error;
+    }
+  });
+};
+
+formButton.addEventListener('submit', submitEvent);
+signOutButton.addEventListener('click', signOut, () => {
+  authHandler.signOut();
+});
```

Comment:
`addEventListener` can't have two `listener`, please reduce it to only one. You can for example call the `signOut` function inside the second `listener`.


<img width="803" alt="image" src="https://github.com/moshoodabdullahi/todo-list-app/assets/123197179/310682fb-3797-4788-8109-c92515612e3f">


See https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

Reply:
Done